src/main/kotlin/assertk/assertions/inputstream.kt

```diff
@@ -0,0 +1,62 @@
+package assertk.assertions
+
+import assertk.Assert
+import assertk.assertions.support.expected
+import java.io.InputStream
+
+fun Assert<InputStream>.hasSameContentAs(other: InputStream) {
+    val buf1 = ByteArray(4096)
+    val buf2 = ByteArray(4096)
+
+    var size = 0L
+    while(true) {
+        val len1 = actual.read(buf1)
```

Comment:
Don't think this test is right. `read` doesn't define how many bytes it will read each time https://docs.oracle.com/javase/7/docs/api/java/io/InputStream.html#read(byte[]). Check out the implementation in apache commons http://commons.apache.org/proper/commons-io/javadocs/api-release/src-html/org/apache/commons/io/IOUtils.html#line.2551